engines/sci/graphics/font.cpp

```diff
@@ -87,13 +91,16 @@ void GfxFontFromResource::draw(uint16 chr, int16 top, int16 left, byte color, bo
 	int y = 0;
 	int16 greyedTop = top;
 
-	byte *pIn = getCharData(chr);
+	SciSpan<const byte> charData = getCharData(chr);
+	if (!charData) {
+		error("Character '%c' does not exist in font %d", chr, _resourceId);
+	}
```

Comment:
Oh, wait, I see now the difference. It would have not actually tried to read from the pointer at all in that case because the character dimensions would be zero.